src/main/java/JavaFx/FixedDurationTask.java

```diff
@@ -0,0 +1,146 @@
+package JavaFx;
```

Comment:
again, check for a better package name that won't conflict with known default names